Update for managing continuous deployment pipelines (#1746)
wso2/docs-choreo-dev: en/docs/devops-and-ci-cd/manage-continuous-deployment-pipelines.md

2. In the Choreo Console, go to the top navigation menu and click **Organization**. Then select your organization.
3. Click the project you want to change the default pipeline.
4. In the left navigation menu, click **DevOps** and then click **CD Pipelines**.
5. Click **Set as Default** corresponding to the pipeline you want to set as the default pipeline for the project. This displays a confirmation dialog that details the impact of setting the new pipeline as the project default.
6. Click **Confirm**.

    !!! info "Note"

>         The **default** continuous deployment pipeline of the organization and the project are two different settings. The organization **default** is used to assign that pipeline to the project at the time of creation. The project **default** is used to show the promotion order of the components that belong to that project in the **Deploy** page.

Reviewer: ```suggestion
        The **default** continuous deployment pipeline at the organization level and the project level are separate settings. The organization's **default** pipeline is assigned to a project when it is created. The project's **default** pipeline determines the default promotion order of its components on the **Deploy** page.
```
